### ISSUE TITLE
Template localization 

### ISSUE THREAD

**Issue by Tajnymag** [egoist/saber.js#13]
I'd like to support different language mutations for my website. Urls would have a domain.com/{lang}/{page} format.

How should I achieve this with Saber.js, if I want to have the pages generated statically?

**Comment by egoist** [comment]
Something like `./routes/en/index.vue` `./routes/jp/index.vue`?

**Comment by Tajnymag** [comment]
Yes, but I meant how should I do this without copying index.vue manually and without having to manage all of them separately.

**Comment by egoist** [comment]
You can use this instead https://github.com/egoist/saber.js#adding-routes-programmatically
